src/Validation/Validator.php

```diff
@@ -189,6 +196,22 @@ public function __construct()
         $this->_providers = self::$_defaultProviders;
     }
 
+    /**
+     * Whether to stop validation rule evaluation on the first failed rule.
+     *
+     * When enabled the first failing rule per field will cause validation to stop.
+     * When disabled all rules will be run even if there are failures.
+     *
+     * @param bool $stopOnFailure If to apply last flag.
+     * @return $this
+     */
+    public function stopOnFailure(bool $stopOnFailure = true)
```

Comment:
My only comment is this interface. We have stopped making function that are positive `stop...` but have a parameter that can make them positive or negative. Can we split this into enable/disable?

Reply:
I find a separated api counter productive. It makes also setting it from a simple input more complex.

Reply:
Then we should call it `set...()` to follow other changes, yes?

Reply:
I changed the method name.